### ISSUE TITLE
CLI morphometrics doesn't generate index images

### ISSUE THREAD

**Issue by hermancollin** [axondeepseg/axondeepseg#553]
**Describe the bug**
Using the `axondeepseg_morphometrics` script and specifying folders with the `-i` option produces an unwanted behavior.
1. we get the following message (which is peculiar because the xlsx file ___is___ saved and look normal)
```bash
Cannot save morphometrics data in file 'paasu_img2_axon_morphometrics.xlsx'.`
```
2. "index" images are not generated

The first point was reported by @mathieuboudreau last week when he tried to run morphometrics via the CLI with subfolders, similar to what is described here.

When running the script directly in the folder containing the segmentation, we do not encounter this problem and the index files are generated as expected. 

**Platform (please complete the following information):**
 - Linux 5.10.52-3-rt47-MANJARO
 - AxonDeepSeg v3.2.0

**Comment by Stoyan-I-A** [comment]
I'm looking at [the section of code that starts here](https://github.com/neuropoly/axondeepseg/blob/443c0579f417f8b1d0046e3904f9d54220b6091e/AxonDeepSeg/morphometrics/launch_morphometrics_computation.py#L216).

It's a bit hard to follow how variables with paths are used. I admit I haven't tested the `-i` option when implementing the index images.
